gitops/snapshot.go

```diff
@@ -696,6 +702,11 @@ func PrepareSnapshot(ctx context.Context, adapterClient client.Client, applicati
 	}
 	snapshot := NewSnapshot(application, &snapshotComponents)
 
+	// expose the source repo URL in the snapshot as annotation do we don't have to do lookup in integration tests
+	if newComponentSource.GitSource != nil {
+		metadata.SetAnnotation(snapshot, SnapshotGitSourceRepoURLAnnotation, newComponentSource.GitSource.URL)
```

Comment:
The [gosec check](https://github.com/konflux-ci/integration-service/actions/runs/9259151522/job/25470527489?pr=747) is complaining about this line not handling the error that's returned by `metadata.SetAnnotation`.